docs/Introduction/Getting Started.md

```diff
@@ -80,17 +225,23 @@ from checkov.terraform.models.enums import CheckResult, CheckCategories
 from checkov.terraform.checks.resource.base_check import BaseResourceCheck
```

Comment:
Let start the sample policy section with terraform snippet before the python snippet.
This way the user will know adding a policy starts at a terraform code base that we would like to scan